### PR TITLE
Add remaining missing keyboard tests

### PR DESCRIPTION
**Pull request by spectranaut**
Add remaining missing toolbar tests! :)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+GITHUB_TOKEN=b263368792c75b349311a742f91c21b5d8691c61
```
Comment (nschonni) [inline .env:1]: Guessing you might want to revoke this token

Comment (spectranaut) [reply .env:1]: llolll thxxx revoking XD

Comment (spectranaut) [reply .env:1]: I knew that would happen XD I have a PR to .gitignore that file

**Comment by spectranaut** [comment]
https://github.com/w3c/aria-practices/pull/1700 should be merged first, because along time ago @maschad96 wrote some tests that never got merged. This PR has some duplicate work which I'll just delete after @maschad96's work is in.

**Comment by spectranaut** [comment]
Ok #1700 is merged so this can be reviewed and merged whenever! I did replace a couple tests where I had already written a test that was a bit more thorough.

**Review by carmacleod — APPROVED** [review]
+1
Nice @spectranaut !
Only a couple of very minor comment typos.
All 49 tests pass! 🎉

**Comment by spectranaut** [comment]
I fixed spelling mistakes, thanks @carmacleod, and I've rebased to resolve merge conflicts!

**Review by mcking65 — APPROVED** [review]
Thank you @spectranaut!!

**Pull request merged**